app/code/core/Mage/Adminhtml/Block/Customer/Edit/Tab/View.php

```diff
@@ -118,11 +113,9 @@ public function getStoreCreateDateTimezone()
      */
     public function getLastLoginDate()
     {
-        $date = $this->getCustomerLog()->getLoginAtTimestamp();
-        if ($date) {
-            return Mage::helper('core')->formatDate($date, Mage_Core_Model_Locale::FORMAT_TYPE_MEDIUM, true);
-        }
-        return Mage::helper('customer')->__('Never');
+        return ($date = $this->getCustomerLog()->getLoginAtTimestamp())
+            ? $this->formatDate($date, Mage_Core_Model_Locale::FORMAT_TYPE_MEDIUM, true, false)
+            : Mage::helper('customer')->__('Never');
```

Comment:
```suggestion
        $date = $this->getCustomerLog()->getLoginAtTimestamp();
        if ($date) {
            return Mage::helper('core')->formatDate($date, Mage_Core_Model_Locale::FORMAT_TYPE_MEDIUM, true);
        }
        return Mage::helper('customer')->__('Never');
```

This is only a style change that i'd like to revert.

Btw ... set variable in condition could trigger code quality tool ... 

 `return ($date = $this->getCustomerLog()->getLoginAtTimestamp())`